kvx/README.md

```diff
@@ -1 +1,114 @@
-See https://crates.io/crates/kvx
+# Key-value store X
+
+Abstraction layer over various key-value store backends in Rust. Tailored to fit the use-cases for [Krill](https://github.com/NLnetLabs/krill).
```

Comment:
> Tailored to fit the use-cases for Krill
But expected to be useful to others too?

Reply:
Free to use as-is